pegen/build.py

```diff
@@ -7,6 +7,7 @@
 from distutils.command.clean import clean  # type: ignore
 from distutils.command.build_ext import build_ext  # type: ignore
 
+from pegen.python_generator import PythonParserGenerator
```

Comment:
This is a missing import (the main script fails to generate the python parser currently) that is unrelated to this PR but was introduced in the same change.